client/src/app/components/CreateProject/Form.js

```diff
@@ -104,10 +104,10 @@ class Form extends React.Component {
                 "Sass"
               ]}
               maxAllowed={5}
-              isDefault="React"
+              isDefault={this.state.stack[0]}
```

Comment:
Personally I'm not sure why we had to introduce the concept of default. From my understanding, it was to avoid pushing empty values to the back end.

However the proper way would have been to have form validation, with certain fields compulsory. But sure, we can leave it like this for now, it's not the first priority

Reply:
We can take the defaults out. It's more for the user experience. I just thought we could have something populated ready for the to change. It's also less choices the user has to make if those defaults are right for them.